ethstaker_deposit/utils/validation.py

```diff
@@ -138,6 +139,14 @@ def validate_deposit(deposit_data_dict: Dict[str, Any], credential: Credential =
 def validate_password_strength(password: str) -> str:
     if len(password) < 8:
         raise ValidationError(load_text(['msg_password_length']))
+
+    encoding = sys.stdin.encoding.lower()
```

Comment:
Can we add unit tests for these, I did a quick pass but you could probably figure out a better approach:

```
def test_validate_password_strength_encoding(monkeypatch):
    class MockStdin:
        def __init__(self, encoding):
            self.encoding = encoding
    mock_stdin = MockStdin(encoding='latin-1')
    monkeypatch.setattr(sys, 'stdin', mock_stdin)
    password = 'oh-nöööö'

    with pytest.raises(ValidationError):
        validate_password_strength(password=password)


def test_validate_password_strength_encoding_windows(monkeypatch):
    class MockStdin:
        def __init__(self, encoding):
            self.encoding = encoding
    mock_stdin = MockStdin(encoding='latin-1')
    monkeypatch.setattr(sys, 'stdin', mock_stdin)
    monkeypatch.setattr(sys, 'platform', 'win32')
    password = 'oh-nöööö'

    with pytest.raises(ValidationError):
        validate_password_strength(password=password)
```

Reply:
Yes, we should. Unit test added.